### PR TITLE
ES6 unit tests

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?
Convert some unit tests that were forgotten when the modules they belong to were converted.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js [hunk 1]
```diff
@@ -3,13 +3,16 @@ import fastdom from 'fastdom';
 import $ from 'lib/$';
 import config from 'lib/config';
 import detect from 'lib/detect';
-import template from 'lodash/utilities/template';
 import trackAdRender from 'commercial/modules/dfp/track-ad-render';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
-import plistaStr from 'raw-loader!commercial/views/plista.html';
 import { loadScript } from 'lib/load-script';
 
-const plistaTpl = template(plistaStr);
```
Comment (gustavpursche) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js]: ❤️

Comment (sndrs) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js]: do we proscribe `lodash/utilities/template` in the lint config? we probably should?

**Comment by shtukas** [comment]
:+1:

File: static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js [hunk 2]
```diff
@@ -3,13 +3,16 @@ import fastdom from 'fastdom';
...
+const plistaTpl = ({ widgetName }) => `
+    <div class="PLISTA" data-ob-template="guardian">
+        <div data-display="plista_widget_${widgetName}"></div>
+    </div>
+`;
+
```
Comment (philwills) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js]: I've spent a fair while failing to get raw-loader to work with Jest. I'm very tempted to follow your lead and inline the template.

**Pull request merged**